client/text.cpp

```diff
@@ -164,14 +165,14 @@ const QString popup_info_text(struct tile *ptile, bool with_links)
   int tile_x, tile_y, nat_x, nat_y;
   bool first;
 
-  std::function<QString(const char *name, help_page_type hpt)> maybe_link;
+  std::function<QString(QString name, help_page_type hpt)> maybe_link;
   if (with_links) {
-    maybe_link = [](const char *name, help_page_type hpt) -> QString {
-      return create_help_link(name, hpt);
+    maybe_link = [](QString name, help_page_type hpt) -> QString {
+      return create_help_link(name.toStdString().c_str(), hpt);
```

Comment:
This could matter in case the system encoding isn't UTF-8 and `std::string` uses a weird encoding:
```suggestion
      return create_help_link(qUtf8Printable(name), hpt);
```

Reply:
Done.